src/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -0,0 +1,66 @@
+<?php namespace October\Rain\Database\Eloquent\Concerns;
+
+use Illuminate\Support\Str;
+
+trait QueriesRelationships
+{
+    use \Illuminate\Database\Eloquent\Concerns\QueriesRelationships;
```

Comment:
Should this be a use or an extend?

Reply:
Traits can't be extended so you have to use "use".

Actually, in this particular case you might argue that the "use" is not necessary at all since the trait I wrote overwrites the `withCount` method on the october model level where the eloquent trait is used on the eloquent model. The tests prove that it still works.

However, I'd opt to leave the "use" statement as it best clarifies what we're doing here (which is override one specific method of the QueriesRelationships trait.

Reply:
Fair enough, although does the "double use" cause any issues with performance or class resolving of any kind?

Reply:
Right, so I've tested the performance with the most popular method I found on google which is simply creating a lot of instances through a for loop. and measure the begin and end time.

I saw no noticeable differences creating 100 000 model instances. All results varied between 0.13 and 0.16 seconds.
If there is a better way to go about this let me know :)

Traits can't be resolved so I can't think of any issues there. 

I walked through the method I implemented again and did notice that the method does use `getRelationWithoutConstraints` and `mergeConstraintsFrom`, both which are part of the "parent" trait.
So removing the use statement could theoratically make the code break in that case (e.g. the eloquent version not beging used). Even if it's unlikely to happen.